src/testup/file_reporter.rb

```diff
@@ -124,7 +130,7 @@ def summary
   def log_basename
     # File system friendly version of ISO 8601. Makes the logs be sortable in
     # the file browser.
-    version = Sketchup.version.split('.').first
+    version = Sketchup.version[/\A\d+/]
```

Comment:
What's the reason for this change? Regexes isn't something everyone is comfortable with. Against I'd argue the original is clearer to read.

Reply:
This isn't a 'hot path', or, it's not called repeatedly.  Hence, speed isn't an issue.

You're implying that Ruby code published 'by Trimble' should be dumbed down for plugin authors.  I think it would be better to show concise, fast, and efficient code, regardless if it appears more complex.  Regex's aren't rocket science...

I'd prefer to leave the Regex, but I could change it to:
```
version, _ = Sketchup.version.split('.', 2)
```